lib/sensu/api/routes/silenced.rb

```diff
@@ -3,6 +3,7 @@ module API
     module Routes
       module Silenced
         SILENCED_URI = /^\/silenced$/
+        SILENCED_ID_URI = /^\/silenced\/ids\/([\w\.\-\*]+:[\w\.\-\*]+)$/
```

Comment:
This feels strange, as the route requires ':' in the query string or the generic 404 will be used. It still accomplishes the same functionality, as accepting any string as an ID will just push 404 responsibilities to the route method.